src/widget/wkey.cpp

```diff
@@ -30,8 +31,11 @@ void WKey::setValue(double dValue) {
             KeyUtils::keyFromNumericValue(dValue);
     if (key != mixxx::track::io::key::INVALID) {
         // Render this key with the user-provided notation.
-        QString keyStr = KeyUtils::keyToString(key);
-        if (m_displayCents) {
+        QString keyStr = "";
+        if (m_displayKey) {
+            keyStr = KeyUtils::keyToString(key);
+        }
+        else if (m_displayCents) {
```

Comment:
is it really "else" here?

Reply:
woopsy..such a small PR, and such a mistake.... fixed.